docs/standard/terminology.md

```diff
@@ -0,0 +1,69 @@
+# Terminology
+
+In order to use key terms consistently across the standard documentation and OCP communication in general, terminology is managed according to a process that ensures both quality and reactivity.
+
+![Terminology process overview](https://www.lucidchart.com/invitations/accept/2eee45ed-5e62-4636-9a44-24b4414f273f)
+
+## 1. Proposition and validation
+
+When new content is added to the documentation and new concepts are introduced in the standard, new terms may need to be translated.
+
+**Reminder:** a term is a word or group of words that must be used and translated consistently to ensure a good understanding and usage of the content. Synonyms of terms must consequently be banned. Expressions that are not terms are called *generic* expressions.
```

Comment:
Re: "Synonyms of terms must consequently be banned." 

I would prefer to say "Synonyms of terms should consequently be **avoided**."  or some rephrase that captures the difference between:

(A) Alternating between a term and its synonyms in the main prose of documentation;

(B) Including synonyms *alongside* a term to clarify its meaning to different audiences. E.g. sometimes in the schema or codelists we might highlight the alternative names a concept goes by in order to help understanding. 

For example,the codelist definition for 'Contract Schedules' states "Any document which contains additional terms, obligations or information related to the contract, such as a schedule, appendix, annex, attachment or addendum." 

These might be thought of as synonyms in some contexts, but we found it important to *name* the specific term that a user might be familiar with.